### ISSUE TITLE
content: LLM 학습 글을 series로 승격 (llm-core, llm-training)

### ISSUE THREAD

**Issue by jwsung91** [jwsung91/jwsung91.github.io#7]
# 배경

LLM 학습 글 7편이 `[LLM-core]` / `[LLM-training]` 제목 접두어로만 묶여 있고 실제 `series` frontmatter는 비어 있다. series로 등록하면:

- 글 하단에 이전/다음 네비게이션(`SeriesNav`)이 자동으로 붙는다.
- 제목에서 대괄호 접두어를 제거할 수 있다 (시리즈명이 맥락을 제공).

# 변경 내용

## 1. 시리즈 2개 신설

**taxonomy 단일화 이슈(#5)가 먼저 완료된 경우**: `src/data/taxonomy.json`의 `series` 배열과 `public/admin/config.yml`의 series options에만 추가.

**아직인 경우** 아래 4곳 모두에 추가:
- `src/content.config.ts`의 `seriesIds`
- `src/lib/series.ts`의 `SERIES_META`
- `scripts/check-content.mjs`의 `seriesMeta` (project 없음 → `{}`)
- `public/admin/config.yml`의 blog 컬렉션 series select options

추가할 값:

| id | title | description | project |
|---|---|---|---|
| `llm-core` | `LLM Core` | `Transformer부터 decoding까지, LLM이 텍스트를 생성하는 핵심 원리를 단계별로 정리합니다.` | 없음 |
| `llm-training` | `LLM Training` | `Pretraining부터 chat model이 되기까지 LLM 학습 과정을 정리합니다.` | 없음 |

## 2. 글 frontmatter 수정 (7편)

파일명은 절대 변경하지 않는다 (파일명 = URL). `title`에서 대괄호 접두어만 제거하고 `series` / `seriesOrder`를 추가한다.

**llm-core** (`src/content/blog/`):

| seriesOrder | 파일 | 새 title |
|---|---|---|
| 1 | `2026-07-02-llm-transformer와-self-attention-이해하기.md` | `Transformer와 self-attention` |
| 2 | `2026-07-02-llm-encoder-only와-decoder-only-구조-이해하기.md` | `Encoder-only와 decoder-only 구조 이해하기` |
| 3 | `2026-07-03-llm-core-tokenizer와-embedding.md` | `Tokenizer와 embedding` |
| 4 | `2026-07-03-llm-core-logit과-softmax.md` | `Logit과 softmax` |
| 5 | `2026-07-03-llm-core-decoding.md` | `Decoding` |

**llm-training**:

| seriesOrder | 파일 | 새 title |
|---|---|---|
| 1 | `2026-07-03-llm-training-pretraining과-fine-tuning.md` | `Pretraining과 fine-tuning` |
| 2 | `2026-07-03-llm-training-base-model은-어떻게-chat-model이-되는가.md` | `Base model은 어떻게 chat model이 되는가` |

주의: 각 파일의 현재 title이 표와 다른 형태라면(접두어 뒤 문구 기준) 접두어만 벗기고 나머지는 원문 유지를 우선한다.

## 3. (선택 후속 — 별도 PR 가능)

`src/pages/blog/index.astro`의 Study Notes 섹션은 현재 study 글 플랫 리스트다. 글이 늘수록 길어지므로, 시리즈에 속한 study 글은 시리즈 카드(제목 + 글 수 + 첫 글 링크)로 묶고 시리즈 없는 study 글만 개별 나열하는 형태로 개선할 수 있다. `src/lib/blog.ts`의 `getFeaturedBlogSeries` / `groupBlogPostsBySeries`(현재 미사용)를 활용 가능.

# 완료 조건

- [ ] 7편 모두 series/seriesOrder가 설정되고 제목에서 대괄호 접두어가 제거됨.
- [ ] 각 글 하단에 SeriesNav(이전/다음)가 렌더됨.
- [ ] 파일명(URL) 변경 없음.
- [ ] `npm run check` 통과 (check-content가 seriesOrder 중복/누락을 검증함).

# 검증

```bash
npm run check
npm run dev   # /blog/2026-07-03-llm-core-decoding 등에서 SeriesNav 확인
```
